pkg/network/transport/object/grpc/replication.go

```diff
@@ -0,0 +1,149 @@
+package object
+
+import (
+	"context"
+	"fmt"
+
+	objectGRPC "github.com/nspcc-dev/neofs-api-go/v2/object/grpc"
+	refs "github.com/nspcc-dev/neofs-api-go/v2/refs/grpc"
+	status "github.com/nspcc-dev/neofs-api-go/v2/status/grpc"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
+)
+
+func (s *Server) Replicate(_ context.Context, req *objectGRPC.ReplicationRequest) (*objectGRPC.ReplicationResponse, error) {
+	const codeInternal = uint32(1024*status.Section_SECTION_FAILURE_COMMON) + uint32(status.CommonFail_INTERNAL)
```

Comment:
dont we have some helpers for this?

Reply:
not for these types

Reply:
can we add them? `1024*status.Section_SECTION_FAILURE_COMMON` looks magic

Reply:
> looks magic

this is how protocol marks the code space. It will only be easier to just write the number - it’s known